drivers/uc8151d/include/uc8151d_params.h

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (C) 2023 Koen Zandberg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_uc8151d
+ *
+ * @{
+ * @file
+ * @brief       Default configuration for the uc8151d
+ *
+ * @author      Koen Zandberg <koen@bergzand.net>
+ */
+
+#ifndef UC8151D_PARAMS_H
+#define UC8151D_PARAMS_H
+
+#include "board.h"
+#include "uc8151d.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#ifndef UC8151D_DISPLAY_X
+/**
+ * @brief Width of the display in pixels.
+ */
+#define UC8151D_DISPLAY_X        (200)
+#endif
+
+#ifndef UC8151D_DISPLAY_Y
+/**
+ * @brief Height of the display in pixels.
+ */
+#define UC8151D_DISPLAY_Y        (200)
```

Comment:
Why is it 200x200 but the display in the video is definitely not a square?

Reply:
*if I remember correctly*, it needs some default here. The driver here is for the uc8151d controller, which can be coupled with some e-paper display. The resolution depends on the attached display, so whatever is here needs to be adjusted almost always with every display and there is not really a good common default.

Reply:
Ah! Then please add `Overwrite this with your display config` or something along those lines. Otherwise, future me will have the same question again!